.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
uncommit this file. 
use git soft reset. then send PR again.

Reply:
removed settings.json with a new commit